### PR TITLE
Add Chinese tranlation for docs/python-types.md

### PR DESCRIPTION
**Pull request by waynerv**

### CODE REVIEW

**Review by RunningIkkyu — APPROVED** [review]
Great!

**Comment by tiangolo** [comment]
Thank you @waynerv for your contribution! :rocket: :bow: 

And thanks @RunningIkkyu for the review/approval! :coffee: :cake: 

I'll just wait for an approval review from someone else before merging.

Maybe @Dustyposa or @phy25 , if you have the chance, could you review this? :pray:

File: docs/zh/docs/python-types.md
Comment (phy25) [inline docs/zh/docs/python-types.md:104]: `"就是它"` is weird here.

**Review by phy25 — APPROVED** [review]
Generally fine. Not sure how high the standard will be, but I saw some weird spaces between text and  `“”（）` which doesn't feel quite right.

File: docs/zh/docs/python-types.md
Comment (waynerv) [reply docs/zh/docs/python-types.md]: Fixed all improper spaces I has found, also improved translation of the line above.

**Comment by tiangolo** [comment]
Awesome! Thank you @waynerv ! :cake: :tada: 

And thanks @phy25 for the review :bow: :clap:

**Pull request merged**
